packages/angular/ssr/src/routes/router.ts

```diff
@@ -6,86 +6,89 @@
  * found in the LICENSE file at https://angular.dev/license
  */
 
+import { ServerAssets } from '../assets';
 import { AngularAppManifest } from '../manifest';
 import { stripIndexHtmlFromURL } from '../utils/url';
 import { getRoutesFromAngularRouterConfig } from './ng-routes';
 import { RouteTree, RouteTreeNodeMetadata } from './route-tree';
 
 /**
  * Manages the application's server routing logic by building and maintaining a route tree.
+ *
+ * This class is responsible for constructing the route tree from the Angular application
+ * configuration and using it to match incoming requests to the appropriate routes.
  */
 export class ServerRouter {
-  /** The route tree used for matching URLs to route metadata. */
-  private routeTree: RouteTree | undefined;
+  /**
+   * Creates an instance of the `ServerRouter`.
+   *
+   * @param routeTree - An instance of `RouteTree` that holds the routing information.
+   * The `RouteTree` is used to match request URLs to the appropriate route metadata.
+   */
+  constructor(private readonly routeTree: RouteTree) {}
```

Comment:
Nit: Should the constructor be private so consumers go through `ServerRouter.from` instead?